index.html

```diff
@@ -5,22 +5,113 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Homework Assignment 3</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,800" rel="stylesheet">
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1>Relaxr</h1>  
+				<nav class="alignright clearfix"> 
```

Comment:
`"clearfix"` here is unnecessary since its immediate child (the `ul`) isn't being floated. The `"clearfix"` just needs to be on the `ul` since the `li`s are being floated